dags/bqetl_release_criteria.py

```diff
@@ -0,0 +1,122 @@
+# Generated via https://github.com/mozilla/bigquery-etl/blob/master/bigquery_etl/query_scheduling/generate_airflow_dags.py
+
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+import datetime
+from utils.gcp import bigquery_etl_query, gke_command
+
+default_args = {
+    "owner": "esmyth@mozilla.com",
+    "start_date": datetime.datetime(2020, 12, 3, 0, 0),
+    "email": ["telemetry-alerts@mozilla.com", "dthorn@mozilla.com"],
+    "depends_on_past": False,
+    "retry_delay": datetime.timedelta(seconds=1800),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+}
+
+with DAG(
+    "bqetl_release_criteria", default_args=default_args, schedule_interval="@daily"
+) as dag:
+
+    release_criteria__dashboard_health__v1 = bigquery_etl_query(
+        task_id="release_criteria__dashboard_health__v1",
+        destination_table="dashboard_health_v1",
+        dataset_id="release_criteria",
+        project_id="moz-fx-data-bq-performance",
```

Comment:
@relud this project and dataset is fine for v1. Should we identify a more permanent home now (e.g., `mozdata`) or validate in the perf project and submit a separate PR later?

Reply:
we don't put actual tables in `mozdata`, just views. This could be moved to `moz-fx-data-shared-prod` though if that's preferable.